pkgs/tools/misc/wayback/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, pkg-config
+, libtool
+, chromium
+}:
+
+buildGoModule rec {
+  pname = "wayback";
+  version = "0.19.1";
+
+  vendorHash = "sha256-TC4uwJswpD5oKqF/rpXqU/h+k0jErwhguT/LkdBA83Y=";
+
+  doCheck = false;
+
+  src = fetchFromGitHub {
+    owner = "wabarc";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-LIWCT0/5T52VQQK4Dy6EFmFlJ02MkfvKddN/O/5zpZc=";
+  };
+
+  nativeBuildInputs = [ pkg-config libtool chromium ];
+
+  meta = with lib; {
+    description = "An archiving tool with an IM-style interface.";
+    homepage = "https://docs.wabarc.eu.org";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
See, https://github.com/wabarc/wayback/blob/4bbf9ea47d04e3e3f67b6516d6e8e867b6ef50a9/LICENSE#L637-L640